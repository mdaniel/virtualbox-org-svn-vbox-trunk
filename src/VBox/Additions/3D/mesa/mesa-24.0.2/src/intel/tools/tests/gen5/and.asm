and(8)          g9<1>.wUD       g9<4>.wUD       524032D         { align16 };
and(8)          g5<1>.xD        g5<4>.xD        1D              { align16 };
and(8)          g5<1>D          g6<8,8,1>D      1D              { align1 };
and(16)         g14<1>D         g12<8,8,1>D     1D              { align1 compr };
and(8)          g2<1>D          g2<8,8,1>UD     1D              { align1 };
and.nz.f0.0(8)  null<1>.xD      g9<4>.xUD       1D              { align16 };
and.nz.f0.0(8)  null<1>D        g8<8,8,1>UD     1D              { align1 };
and(8)          g8<1>UD         g2.4<0,1,0>UD   0x80000000UD    { align1 };
and.nz.f0.0(16) null<1>D        g12<8,8,1>UD    1D              { align1 compr };
and(16)         g12<1>UD        g2.4<0,1,0>UD   0x80000000UD    { align1 compr };
and(16)         g6<1>D          g4<8,8,1>UD     1D              { align1 compr };
and(8)          g2<1>UD         g4<8,8,1>UD     g3<8,8,1>UD     { align1 };
and(16)         g6<1>UD         g8<8,8,1>UD     g4<8,8,1>UD     { align1 compr };
and(8)          g17<1>.xUD      g1<0>.xUD       0x80000000UD    { align16 };
and(8)          g47<1>.xUD      g48<4>.xUD      g47<4>.xUD      { align16 };
and(1)          g8<1>UD         f0<0,1,0>UW     0x0000000fUD    { align1 nomask };
and.nz.f0.0(8)  g7<1>D          g7<8,8,1>D      1D              { align1 };
and.nz.f0.0(16) g14<1>D         g8<8,8,1>D      1D              { align1 compr };
and(8)          g3<1>.xD        g3<4>.xUD       1D              { align16 };
and.nz.f0.0(1)  null<1>UD       g1.6<0,1,0>UD   0x04000000UD    { align1 };
