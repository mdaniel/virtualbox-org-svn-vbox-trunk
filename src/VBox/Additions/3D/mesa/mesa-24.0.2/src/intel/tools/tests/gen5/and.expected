05 01 60 00 21 1c 28 21 2f 01 6f 00 00 ff 07 00
05 01 60 00 a5 1c a1 20 a0 00 60 00 01 00 00 00
05 00 60 00 a5 1c a0 20 c0 00 8d 00 01 00 00 00
05 20 80 00 a5 1c c0 21 80 01 8d 00 01 00 00 00
05 00 60 00 25 1c 40 20 40 00 8d 00 01 00 00 00
05 01 60 02 24 1c 01 20 20 01 60 00 01 00 00 00
05 00 60 02 24 1c 00 20 00 01 8d 00 01 00 00 00
05 00 60 00 21 0c 00 21 50 00 00 00 00 00 00 80
05 20 80 02 24 1c 00 20 80 01 8d 00 01 00 00 00
05 20 80 00 21 0c 80 21 50 00 00 00 00 00 00 80
05 20 80 00 25 1c c0 20 80 00 8d 00 01 00 00 00
05 00 60 00 21 04 40 20 80 00 8d 00 60 00 8d 00
05 20 80 00 21 04 c0 20 00 01 8d 00 80 00 8d 00
05 01 60 00 21 0c 21 22 20 00 00 00 00 00 00 80
05 01 60 00 21 04 e1 25 00 06 60 00 e0 05 60 00
05 02 00 00 01 0d 00 21 00 06 00 00 0f 00 00 00
05 00 60 02 a5 1c e0 20 e0 00 8d 00 01 00 00 00
05 20 80 02 a5 1c c0 21 00 01 8d 00 01 00 00 00
05 01 60 00 25 1c 61 20 60 00 60 00 01 00 00 00
05 00 00 02 20 0c 00 20 38 00 00 00 00 00 00 04
